main.cpp

```diff
@@ -593,24 +210,17 @@ int main()
 
     std::cout << "- ";
     std::cout << "FloatType UDT calculations:\n";
-    ft.add(it).multiply(ft).subtract(dt).divide(it);
+    ft.add(it).multiply(ft).subtract(static_cast<float>(dt)).divide(it);
```

Comment:
- [ ] revise this to say: 
```
std::cout << "FloatType UDT calculations: " << ft.add(it).multiply(ft).subtract(static_cast<float>(dt)).divide(it) << "\n";
```
This will cause the conversion functions to be invoked by the std::cout statement.